enterprise/api_client/discovery.py

```diff
@@ -121,6 +121,29 @@ def get_search_results(self, querystring=None, traverse_pagination=True):
             many=False,
         )
 
+    def get_catalog_results(self, query):
+        """
+        Return results from the discovery service's search/all endpoint.
```

Comment:
Return results from the discovery service's search/all endpoint using POST method.